quidel_covidtest/delphi_quidel_covidtest/data_tools.py

```diff
@@ -166,7 +166,8 @@ def raw_positive_prop(positives, tests, min_obs):
     # nan out any days where there are insufficient observations
     # this also elegantly sidesteps 0/0 division.
     tests[tests < min_obs] = np.nan
-    positive_prop = positives / tests
+    # Jeffreys Correction for estimates
+    positive_prop = (positives+0.5) / (tests+1)
     se = np.sqrt(_prop_var(positive_prop, tests))
```

Comment:
Should this be `tests+1`?

Reply:
Do you mean the denominator in se? I don't think so. According to what we have in [Doctor Visits](https://github.com/cmu-delphi/covidcast-indicators/blob/main/doctor_visits/delphi_doctor_visits/sensor.py#L238), only the proportional estimates are corrected. Since they are used as p in se, se is then corrected too.